users/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+class User(models.Model):
+    nickname     = models.CharField(max_length = 20, unique = True)
+    email        = models.EmailField(unique = True)
+    password     = models.CharField(max_length = 100)
+    phone_number = models.CharField(max_length= 11, null = True)
+    avatar       = models.URLField(null = True)
```

Comment:
```suggestion
    avatar_image       = models.URLField(max_length=2000, null = True)
```
아바타 이미지인지 어떤것인지 알수 있는 네이밍을 해주세요.

Reply:
넵 알겠습니다 😀